app/views/courses/_create_new_course.html.erb

```diff
@@ -1,20 +1,26 @@
 <h3>Create New Course</h3>
 
 <%= form_for :newCourse, url: courses_path, builder: FormBuilderWithDateTimeInput do |f| %>
-  <%= f.error_messages %>
-
-  <%= f.text_field :name, required: true,
+  <%= f.text_field :name, size: 3, required: true,
```

Comment:
I don't think `size` attribute is needed here, it just sets the width of the text field but the text field spans the width of the entire page